maker/src/main.rs

```diff
@@ -123,6 +127,21 @@ async fn main() -> Result<()> {
     let db =
         sqlite_db::connect(data_dir.join("maker.sqlite"), opts.ignore_migration_errors).await?;
 
+    let blocked_peers_path = data_dir.join("blocked_peers.toml");
+    let blocked_peers = match load_blocked_peers(&blocked_peers_path).await {
+        Ok(loaded) => loaded,
+        Err(error) => {
+            tracing::error!(
+                ?blocked_peers_path,
+                %error,
+                "Error loading blocked peers list; ignoring and allowing all connections",
+            );
+
+            HashSet::default()
+        }
+    };
```

Comment:
nit: perhaps a bit more idiomatic (feel free to push back if it's too subjective)

```rust
    let blocked_peers = load_blocked_peers(&blocked_peers_path).unwrap_or_else(|error| {
        tracing::error!(
            ?blocked_peers_path,
            %error,
            "Error loading blocked peers list; ignoring and allowing all connections",
        );
        HashSet::default()
    });
```